include/micm/solver/rosenbrock.inl

```diff
@@ -132,7 +157,35 @@ namespace micm
       for (auto& name : system.UniqueNames(state_reordering))
         variable_map[name] = index++;
     }
-    
+
+    MatrixPolicy<double> absolute_tolerances(parameters_.number_of_grid_cells_, variable_map.size(), 1e-3);
+    if (parameters_.absolute_tolerance_.size() != absolute_tolerances.AsVector().size())
```

Comment:
Why are the sizes of these vectors being compared?

Reply:
in case someone set them on the parameters before the simulation because they want to override the values